### PR TITLE
fix: Add FilipinoShopeeReviewsClassification

### PR DESCRIPTION
**Pull request by ljvmiranda921**
## Checklist for adding MMTEB dataset

Reason for dataset addition:

Add more support for fil/tag language

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [points folder](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

File: mteb/tasks/Classification/fil/FilipinoShopeeReviewsClassification.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            "revision": "d096f402fdc76886458c0cfb5dedc829bea2b935",
+        },
+        type="Classification",
+        task_subtypes=["Sentiment/Hate speech"],
+        category="s2s",
+        eval_splits=["validation", "test"],
+        eval_langs=["fil-Latn"],
+        form=["written"],
+        domains=["Social"],
+        license=None,
```
Comment (shreeya-dhakal) [inline mteb/tasks/Classification/fil/FilipinoShopeeReviewsClassification.py:21]: license is mpl-2.0

**Comment by ljvmiranda921** [comment]
Thanks for the reviews @shreeya-dhakal ! Updated :)

**Review by isaac-chung — APPROVED** [review]
Looks good! Please also add points for review.

**Comment by ljvmiranda921** [comment]
Updated! Thanks @isaac-chung and @shreeya-dhakal for your reviews :)

**Pull request merged**